python/hail/context.py

```diff
@@ -454,13 +484,24 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
             wrong number of elements to missing, rather than setting
             the entire genotype to missing.
 
+        :param bool generic: If True, read the genotype with a generic schema.
+
+        :param call_fields: FORMAT fields in VCF to treat as a :py:class:`~hail.type.TCall`. Only applies if ``generic=True``.
+        :type call_fields: str or list of str
```

Comment:
We do this all over the place. Seems like it's only a problem if we actually mutate the list -- it's not really possible for a user to do that. Since we just pass them into the JVM, it should be OK.